translations/translations.json

```diff
@@ -7054,6 +7054,7 @@
       "settings": "Einstellungen",
       "copyUrl": "Link kopieren",
       "contrastTheme": "Kontrast Modus",
+      "contrastThemeDescription": "Mit dem Kontrastmodus wird die Lesbarkeit Ihrer Anwendung verbessert",
```

Comment:
🔧 We use `du` everywhere in the app. Perhaps the following would be even simpler though:
```suggestion
      "contrastThemeDescription": "Mit dem Kontrastmodus wird die Lesbarkeit verbessert",
```

Reply:
got you :+1: